boards/seeed/xiao_esp32c6/xiao_esp32c6-pinctrl.dtsi

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 Mario Paja
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32c6-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32c6-gpio-sigmap.h>
+
+&pinctrl {
+
+	uart0_default: uart0_default {
```

Comment:
do you want to leave `uart0` enabled and connected to these pins?
I couldn't be sure looking at the schematic, but it seemed these pins are unavailable

Reply:
Hi,
In the schematics the used gpios are rather unclear. The best is to check the pinout file. 

Yes, I think of leaving uart enabled like the rest of the xiao_esp32xx lineup.

![image](https://github.com/user-attachments/assets/47e1e8c0-0edc-4caf-91dc-7ad8fcaac04a)